man/neptune_fetch.Rd

```diff
@@ -0,0 +1,42 @@
+\name{neptune_fetch}
```

Comment:
Doc string should be based on this one including examples
https://github.com/neptune-ai/neptune-client/blob/39de77f1aa3f6fd3095e6aca84e7c8254e4dea58/neptune/new/attribute_container.py#L197

Reply:
@needlecapter this one was missed